apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/sharing-modal/index.tsx

```diff
@@ -65,9 +71,12 @@ const SharingModalInner: React.FC = () => {
 	const shouldShowVideoCelebrationModal =
 		useShouldShowVideoCelebrationModal( isCurrentPostPublished );
 
-	const [ isOpen, setIsOpen ] = useState( false );
+	const [ isOpen, setIsOpen ] = useState( true );
 	const closeModal = () => setIsOpen( false );
 	const { createNotice } = useDispatch( noticesStore );
+	const [ shouldShowSuggestedTags, setShouldShowSuggestedTags ] = React.useState(
+		postMeta?.reader_suggested_tags?.length > 0
```

Comment:
Still to test, but reading through this stands out to me as potentially buggy.

If we set the initial state based on this, then say for the case of a new post `shouldShowSuggestedTags` would be set as `false` when first loaded and we have no other effects that toggle this to `true` after saves may update this post meta value to contain suggestions? So the SuggestedTags component would never render since this value would still be false.

Alternatively, we could assume `true` from the beginning. Once the modal toggles to be opened and the `SuggestedTags` component mounts, we could use its check for the post meta there to toggle this to `false` if there are no suggestions present. That would be evaluating the presence of suggested tags at the time of publish as opposed to at the time of opening the editor.